packages/@okta/vuepress-site/docs/reference/api/rulesets/index.md

```diff
@@ -0,0 +1,979 @@
+          "type": "Okta:SignOn",
+          "status": "ACTIVE",
+          "default": false,
+          "_links": {
+              "mappings": {
+                  "href": "https://your-org.okta.com/api/v1/policies/rst30qv3igD5OpiL50g7/mappings",
+                  "hints": {
+                      "allow": [
+                          "GET"
+                      ]
+                  }
+              },
+              "self": {
+                  "href": "https://your-org.okta.com/api/v1/policies/rst30qv3igD5OpiL50g7",
+                  "hints": {
+                      "allow": [
+                          "GET",
+                          "PUT",
+                          "DELETE"
+                      ]
+                  }
+              },
+              "rules": {
+                  "href": "https://your-org.okta.com/api/v1/policies/rst30qv3igD5OpiL50g7/rules",
+                  "hints": {
+                      "allow": [
+                          "GET"
+                      ]
+                  }
+              },
+              "deactivate": {
+                  "href": "https://your-org.okta.com/api/v1/policies/rst30qv3igD5OpiL50g7/lifecycle/deactivate",
+                  "hints": {
+                      "allow": [
+                          "POST"
+                      ]
+                  }
+              },
+              "applications": {
+                  "href": "https://your-org.okta.com/api/v1/policies/rst30qv3igD5OpiL50g7/app",
+                  "hints": {
+                      "allow": [
+                          "GET"
+                      ]
+                  }
+              }
+          }
+      }
+```
+
+### Rule Set object
+
+The Rule Set object defines several attributes:
+
+| Parameter   | Description                                                                                          | Data Type                                         | Required | Default                |
+| ---------   | -----------                                                                                          | ---------                                         | -------- | -------                |
+| id          | Identifier of the rule set                                                                           | String                                            | No       | Assigned               |
+| type        | Specifies the [type of rule set](#rule-set-types). Valid values: `Okta:SignOn` or `Okta:UserProfile` | String (Enum)                                     | Yes      |                        |
+| name        | Name of the rule set                                                                                 | String                                            | Yes      |                        |
+| system      | This is automatically set to `true` for system policies, which cannot be deleted.                    | Boolean                                           | No       | `false`                |
```

Comment:
Was `system` changed to `default`? The JSON doesn't contain a property with this name.

Reply:
Yes